### DISCUSSION TITLE
What is the difference between the CPU boost modes e.g. Enabled, Efficient Enabled, Efficient Aggressive, Aggressive?

### DISCUSSION THREAD

**Discussion by Metalogic999** [seerge/g-helper#492] [Q&A] [answered]
What is the difference between the CPU boost modes e.g. Enabled, Efficient Enabled, Efficient Aggressive, Aggressive? I've currently just left Performance and Turbo mode on "Efficient Aggressive" and everything seems to work quite well, but I don't really know what it's doing...

What are the different modes actually doing and are some better for some scenarios than others? Some games I run are very CPU intensive, others less so, but more graphically demanding

**Comment by seerge** [comment, marked as answer]
I think this https://www.reddit.com/r/WindowsHelp/comments/u1egv5/differences_in_cpu_performance_boost_modes_in_the/?utm_source=share&utm_medium=ios_app&utm_name=ioscss&utm_content=1&utm_term=1

**enabled**: Just tries to boost a little more out of your CPU, if your CPU is hot or some problem occurs, well it's back to low GHz

**aggressive**: Just tries to boost a little more out of your CPU, if your CPU is hot or some problem occurs, it will make the fans go faster to get more out of your CPU. also this consumes more power EVEN when idling and your system is doing nothing.

**enabled efficient**: It will do the same thing as enabled only thing is, it might try to use less power

**aggressive efficient**: It will do the same thing as aggressive although try to use as less power as possible

**Reply by Metalogic999** [reply]
Thanks, makes sense. Think I'll leave it on "Efficient Aggressive" for now, except for silent mode (CPU boost off), maybe "Efficient Enabled" for "Balanced"

**Reply by mjishnu** [reply]
@seerge hello thanks for the answer i was also looking for this, what about “aggressive at guaranteed” and “efficient aggressive at guaranteed” options?

**Reply by seerge** [reply]
I think they are more or less the same thing for modern devices

**Reply by mjishnu** [reply]
thanks for the quick reply. just out of curiosity, do you know what this "at guaranteed" does it means it would draw power at a more consistent rate or something

**Reply by seerge** [reply]
@m-jishnu quick googling has led me to https://www.reddit.com/r/MSILaptops/comments/d90kyt/processor_performance_boost_mode_tests/ :)

**Reply by mjishnu** [reply]
thanks again for the reply btw, i have a questions its kinda dumb but in ghelper does green means there is a new update and greyed out means no update ? 
<img width="1094" alt="image" src="https://github.com/seerge/g-helper/assets/83004520/973baa3d-6c3e-432c-a30b-ab6f0106e732">
also does it fetch hal updates like the new one which add dynamic lighting support for rog devices
<img width="1280" alt="image" src="https://github.com/seerge/g-helper/assets/83004520/b8a00a90-436b-41d0-be69-2b3acfc785c1">

**Reply by seerge** [reply]
@m-jishnu i was answering this before

Green means you have same or newer version that on website (you are good), gray means it doesn't know your local version (or you don't have that hardware at all , like intel wifi adapter or so)

As for AC updates list - it is not a drivers list, it's just own components of AC itself, that you don't need if you don't have AC

**Reply by mjishnu** [reply]
I see, one last question so if i completely nuke all Asus/armory crate service and software g helper would work fine right is independent of Asus service, i am going to format my pc so i don't need to install these bloats again do i. also sorry for pestering you with questions 😅

**Reply by seerge** [reply]
It doesn't depend on anything from asus

**Reply by Zahanboi** [reply]
> It doesn't depend on anything from asus

Hello sir, I just wanna know that is it safe to temper with cpu boost settings, i want my cpu to last longest

**Reply by seerge** [reply]
@Zahanboi cpu boost is just a setting in windows power plan :) You can select anything you want there.
